shared/reducers/chat2.js

```diff
@@ -735,15 +750,46 @@ const rootReducer = (
         }
       )
 
+      let containsLatestMessageMap = state.containsLatestMessageMap.withMutations(map => {
+        Object.keys(convoToMessages).forEach(cid => {
+          const conversationIDKey = Types.stringToConversationIDKey(cid)
+          if (map.get(conversationIDKey, false)) {
+            return
+          }
+          const meta = state.metaMap.get(conversationIDKey, null)
+          const ordinals = messageOrdinals.get(conversationIDKey, I.OrderedSet()).toArray()
+          let maxMsgID = 0
+          for (let i = ordinals.length - 1; i >= 0; i--) {
+            const ordinal = ordinals[i]
+            const message = messageMap.getIn([conversationIDKey, ordinal])
+            if (message && message.id > 0) {
+              maxMsgID = message.id
+              break
+            }
+          }
+          if (meta && maxMsgID >= meta.maxVisibleMsgID) {
+            map.set(conversationIDKey, true)
+          }
+        })
+      })
+
       return state.withMutations(s => {
         s.set('messageMap', messageMap)
+        s.set('containsLatestMessageMap', containsLatestMessageMap)
         // only if different
         if (!state.messageOrdinals.equals(messageOrdinals)) {
           s.set('messageOrdinals', messageOrdinals)
         }
         s.set('pendingOutboxToOrdinal', pendingOutboxToOrdinal)
       })
     }
+    case Chat2Gen.jumpToRecent:
+      return state.setIn(['containsLatestMessageMap', action.payload.conversationIDKey], true)
```

Comment:
not sure this is true

Reply:
All this action does it load the most recent messages, so it has to be true.